.github/workflows/release.yml

```diff
@@ -0,0 +1,164 @@
+    needs: [define-gcs-versions-to-update]
+    strategy:
+      matrix:
+        sdk-version: ${{ fromJSON(needs.define-gcs-versions-to-update.outputs.sdk-versions) }}
+    runs-on: ubuntu-latest
+    env:
+      gcp-project-id: 118585658141
+      service-account: gh-ci-optable-web-sdk
+    steps:
+      - name: Download web artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: dist-web
+          path: browser/dist
+
+      - uses: "google-github-actions/auth@v2"
+        with:
+          workload_identity_provider: "projects/${{ env.gcp-project-id }}/locations/global/workloadIdentityPools/optable-ci/providers/github-pool-provider"
+          service_account: "${{ env.service-account }}@optable-platform-ci.iam.gserviceaccount.com"
+
+      - name: Upload SDK to GCS bucket, upload new version
+        uses: "google-github-actions/upload-cloud-storage@v2"
+        with:
+          path: "browser/dist/sdk.js"
+          destination: "gs://optable-web-sdk/${{ matrix.sdk-version }}"
+
+  deploy-demo:
+    needs: [deploy-sdk-to-npm]
+    permissions:
+      contents: "read"
+      id-token: "write"
+    runs-on: ubuntu-latest
+    env:
+      gcp-project-id: 118585658141
+      service-account: gh-ci-optable-web-sdk
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Get sdk artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: dist-web
+          path: browser/dist
+
+      - name: Get vanilla-demo artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: dist-demo-vanilla
+          path: demos
+
+      - name: Get react-demo artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: dist-react-demo
+          path: demos/react/dist
+
+      - name: Get npm-demo artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: dist-npm-demo
+          path: demos/npm/dist
+
+      - uses: "google-github-actions/auth@v2"
+        id: auth
+        with:
+          workload_identity_provider: "projects/${{ env.gcp-project-id }}/locations/global/workloadIdentityPools/optable-ci/providers/github-pool-provider"
+          service_account: "${{ env.service-account }}@optable-platform-ci.iam.gserviceaccount.com"
+
+      - name: Build web-sdk-demos Docker Image
+        run: |
+          DOCKER_BUILDKIT=1 docker build \
+            -t us-docker.pkg.dev/optable-artifact-registry/optable/optable-web-sdk-demos:${{ github.ref_name }} \
+            ./demos
+
+      - name: Login to Google Artifact Registry
+        uses: docker/login-action@v3
+        with:
+          registry: us-docker.pkg.dev
+          username: oauth2accesstoken
+          password: ${{ steps.auth.outputs.auth_token }}
+
+      - name: Publish web-sdk-demos to us-docker.pkg.dev
+        run: docker push us-docker.pkg.dev/optable-artifact-registry/optable/optable-web-sdk-demos:${{ github.ref_name }}
```

Comment:
This step will likely fail since the service account does not have permission to push to GAR.
We will have to give the SA the write permission by updating this file: https://github.com/Optable/optable-terraform/blob/master/production/optable-artifact-registry/locals_common.tf#L16

Reply:
Good catch, PR opened: https://github.com/Optable/optable-terraform/pull/5860